staging/src/k8s.io/apiserver/pkg/storage/etcd3/size_monitor.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package etcd3
```

Comment:
Moved this into the `etcd3` package mostly for "symmetry" with e.g. `etcd3.NewLatencyTracker` and `etcd3.StartCompactor`.

Reply:
moves in their own commit would make review much easier.

Reply:
Agreed - this move was a little late in the game. I'll attempt to break it out into a separate commit but until I do you have my word that there's no changes to the logic that moved in this file. 😄